OpenEXR/IlmImf/ImfDwaCompressor.cpp

```diff
@@ -2491,7 +2491,7 @@ DwaCompressor::uncompress
     // start of the data block.
     //
 
-    if ((version < 0) || (version > 2))
+    if (version > 2)
```

Comment:
This should be fine -- we're just testing for bogus values read from the file. Garbage should show up as >2 if we're interpreting the values as unsigned.